aries-site/src/examples/components/rangeinput/RangeInputExample.js

```diff
@@ -5,7 +5,12 @@ export const RangeInputExample = () => {
   const [value, setValue] = useState(80);
 
   return (
-    <Box direction="row" gap="medium" width="large">
+    <Box
+      pad={{ horizontal: '11px', vertical: '5px' }}
+      direction="row"
```

Comment:
alignment of labels and range
![Screen Shot 2021-09-22 at 9 14 23 AM](https://user-images.githubusercontent.com/1756948/134372347-e6868cae-04dd-48cd-9f7b-e95454905674.png)

Reply:
good catch I fixed that!